arcanist-user.rst

```diff
@@ -243,10 +243,25 @@ shows the current stack of revisions:
    :align: center
    :alt: Screenshot of a revision stack
 
-You can also add ``Depends on D<revision ID>`` to the child's commit
-message, replacing ``<revision ID>`` with the ID of the parent
+You can also add ``Depends on D<revision ID>`` to the child's revision
+summary, replacing ``<revision ID>`` with the ID of the parent
```

Comment:
My only hesitation here is that the revision summary is created from the commit message. I fear that not mentioning the commit message will cause confusion, e.g. "how do I put stuff into the revision summary??".

Reply:
I share that confusion.  Can we define "revision summary" "commit message 'and "commit summary"?

Reply:
In this doc, you mean?

Reply:
At least in the PR comments -- I don't understand the distinction you're making.

Reply:
Ah okay:

- **commit message**: the entire message in the commit, which appears in the full {hg, git} logs
- **commit summary**: the first line of a commit message (I don't think this is an official definition, but we often refer to it. Also I am just now realizing this is confusing given the next definition.)
- **revision summary**: the field in a Phabricator revision within the "Details" panel that is labelled "Summary". This is also editable as part of the ``arc diff`` submission process.

Also,
- **revision title**: the top, bold line in a revision, which you would also see in a list of revisions along with the revision ID

When a revision is created via Arcanist, the editor that pops open defaults to setting the revision title from the commit summary and the revision summary from the rest of the commit message. Editing those fields at this point will also update the commit message (if history is set as mutable, which is a Phabricator setting we generally apply to our repos). 

So, long story short, it doesn't _really_ matter if you put the ``Depends on ...`` into the commit message or the revision summary, as it will be mirrored to the other when the revision is submitted via ``arc diff``.

Reply:
OK, so it sounds like no change is needed here.  Thanks!